swarm/storage/dpa_test.go

```diff
@@ -39,17 +44,12 @@ func TestDPArandom(t *testing.T) {
 		memStore: memStore,
 		DbStore:  db,
 	}
-	chunker := NewTreeChunker(NewChunkerParams())
-	dpa := &DPA{
-		Chunker:    chunker,
-		ChunkStore: localStore,
-	}
-	dpa.Start()
-	defer dpa.Stop()
+
+	dpa := NewDPA(localStore, NewDPAParams())
 	defer os.RemoveAll("/tmp/bzz")
```

Comment:
Shouldn't this be `defer os.RemoveAll(tdb.dir)`? I know that this is the old code, but this test may be improved here as well.